app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -307,67 +307,20 @@ void main() {
       githubGraphQLClient.verifyMutations(<MutationOptions>[]);
     });
 
-    test('Does not merge PR with in progress checks', () async {
+    test('Does not merge PR with neutral check suite', () async {
       branch = 'pull/0';
       final PullRequestHelper prInProgress = PullRequestHelper(
-        lastCommitCheckRuns: const <CheckRunHelper>[
-          CheckRunHelper.windowsInProgress,
-        ],
+        checkSuiteConclusion: CheckRunConclusion.neutral,
       );
       flutterRepoPRs.add(prInProgress);
       await tester.get(handler);
       _verifyQueries();
       githubGraphQLClient.verifyMutations(<MutationOptions>[]);
     });
 
-    test('Does not merge PR with queued checks', () async {
```

Comment:
Why are we deleting these tests?

Reply:
Most of these are duplicates as conclusions are simpler than scanning all the checks ourselves. I added an explicit test to ensure null checksuite conclusion does not submit the pr.

https://docs.github.com/en/rest/guides/getting-started-with-the-checks-api#about-check-suites